tests/helm/terraform/static/all_fields.output.yaml

```diff
@@ -298,18 +298,40 @@ data:
         done
     fi
   
-    # Fix URL to remove "v1" or "v1/"
-    export SUMOLOGIC_BASE_URL=${SUMOLOGIC_BASE_URL%v1*}
+    function fix_sumo_base_url() {
+      local BASE_URL=$SUMOLOGIC_BASE_URL
```

Comment:
@starzu-sumo Sorry for another comment, I was just checking before merge
Didn't lint complain about this line? :fearful:

Reply:
I was thinking rather about `"${SUMOLOGIC_BASE_URL}"`

Reply:
but I believe both changes should be added